server/app.js

```diff
@@ -14,19 +15,23 @@ import {
 const apiRoot = "/api";
 const studentRoot = "/api/student";
 
+
 const app = express();
 
 app.use(express.json());
 app.use(configuredHelmet());
 app.use(configuredMorgan());
 
+
+
 if (config.production) {
 	app.enable("trust proxy");
 	app.use(httpsOnly());
 }
 
 app.use(apiRoot, apiRouter);
 app.use(studentRoot, studentRouter);
+app.use("/api/auth", studentAuthRouter);
 app.use("/health", (_, res) => res.sendStatus(200));
 app.use(clientRouter(apiRoot));
 app.use(clientRouter(studentRoot));
```

Comment:
Hi C, not sure if you did, but i think you should add - app.use(clientRouter(studentAuthRoot)) which is linked to the variable on line 17